packages/cli/src/commands/update.ts

```diff
@@ -0,0 +1,98 @@
+import axios from "axios";
+import inquirer from "inquirer";
+import semver from "semver";
+import { readFileSync, writeFileSync } from 'fs';
+import { logger } from "../utils/logger.js";
+import { execSync } from "child_process";
+
+export async function updateCommand() {
+    try {
+        const installedPackages = JSON.parse(execSync('npm list --depth=0 --json').toString());
```

Comment:
Can you do this in a package-manager agnostic way to support `npm`, `yarn`, and `pnpm`? You can see we do something similar here:

https://github.com/triggerdotdev/trigger.dev/blob/2d722b94bd448e39506f0d6c9fff3c83e373ef2b/packages/cli/src/utils/installDependencies.ts#L7

Also could you use a promise version of `exec` instead of `execSync`? If there is one in the Node.js standard library then don't worry about it.

Reply:
switched to execa

Reply:
switched to exec